rules/damage.md

```diff
@@ -29,9 +29,9 @@ lists the abbreviations:
 
 ## Combined damage types
 
-Some weapons or effects indicate two damage types. For example, a plasma sword could be described as dealing "5d8 E & F".
+Some weapons or effects indicate two damage types. For example, a plasma sword could be described as dealing "5d8 F & E".
 When dealing damage with that weapon, roll 5d8, and half the result (rounded down) will be the first kind of damage listed
-(i.e. electricity), and the remaining damage will be fire. If you rolled a total of 25, that means the target will receive
+(i.e. fire), and the remaining damage will be electricity. If you rolled a total of 25, that means the target will receive
 12 points of fire damage and 13 of electricity. Note thats those damages will go separately through any resistance/immunity
 that the target may have (for example, if the target is resistant to electricity and vulnerable to fire, it will take 24 points
 of damage due to the fire, and 6 due to the electricity)
```

Comment:
Here I changed the weapon description to match the example below.
But I have doubts if there was an error with the example or if the explanation was mistaken, and it should say `"and half the result (rounded **up**) will be the first kind of damage listed"`

Reply:
There's an error (it's 12 electric, 13 fire). However I would leave the example as `E & F` rather than `F & E`, because the former is how most plasma weapons are described in the [weapon tables](https://dmoisset.github.io/s5/equipment/weapon-tables.html), and plasma is by far the most common example of weapons with two damage types